docs/design.md

```diff
@@ -119,3 +119,9 @@ and most of the validator teams have trouble managing logs from running nodes du
 ![monitoring_to_be](https://user-images.githubusercontent.com/6308023/210969235-4aa505ee-28cc-4e16-8129-843dbc4f2ca0.png)
 
 **VATZ** will support to more monitoring and analysis 3rd party apps tool as shown in the diagram above.
+
+---
+
+# VATZ Project Design (v2)
+The comprehensive design of VATZ v2 is presently in the development stage, with an anticipated release scheduled for 2024. 
```

Comment:
currently in development

Reply:
@qkrybin 
I think it's the pretty same phrase as 
currently in development = presently in the development stage
what do you think?

Reply:
agree